src/utils/__tests__/app-storage-test.js

```diff
@@ -0,0 +1,25 @@
+import { AppStorage } from "../app-storage";
+
+describe("AppStorage", () => {
+  const appStorage = new AppStorage();
+  test("AppStorage setItem", () => {
```

Comment:
can you add a test that sets localStorage to be null and then verify that get/set a value works now with cookies